test/SilentEchoTest.ts

```diff
@@ -16,7 +16,7 @@ describe("SilentEcho", function() {
 
     describe("#message()", () => {
         it("Should return a transcript", async () => {
-            const sdk = new SilentEcho(token);
+            const sdk = new SilentEcho(process.env.TEST_TOKEN || "");
```

Comment:
I ran into this error also, though only on circle....do you know what is causing it? I don't get it locally. I updated my version of typescript, but still does not occur locally....maybe my tslint version? I'm at a loss